app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,12 @@
       <%= link_to "All Items", "/items"%>
       <%= link_to "Cart: #{cart.total_items}", "/cart" %>
 
-    <% if current_user %>
+    <% if current_employee? %>
```

Comment:
I think this is a great direction! I expect we will need to add another elsif for admin